### PR TITLE
fix: add primary attribute for current user

### PR DESCRIPTION
**Pull request by luka-nextcloud**
Signed-off-by: Luka Trovic <luka@nextcloud.com>

* Resolves: #2838
* Target version: master

### CODE REVIEW

**Review by max-nextcloud — COMMENTED** [review]
The approach looks good.
ESLint has some complains as can be seen in the Files changed tab.
Other than that seems good to go.

File: src/extensions/Mention.vue [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <template>
 	<NodeViewWrapper as="span" class="mention" contenteditable="false">
-		<NcUserBubble :user="node.attrs.id" :display-name="username" class="mention-user-bubble">
+		<NcUserBubble :user="node.attrs.id" :display-name="username" :primary="isCurrentUser()" class="mention-user-bubble">
```
Comment (vinicius73) [inline src/extensions/Mention.vue:3]: Could you try to avoid call functions inside templates?
Maybe computed works, `this.node` is reactive.

**Comment by luka-nextcloud** [comment]
@max-nextcloud @vinicius73 Please check again.

**Review by vinicius73 — APPROVED** [review]

**Comment by mejo-** [comment]
I added a commit to fix the (unrelated) eslint warning introduced by #2867.

**Review by mejo- — APPROVED** [review]

**Pull request merged**